src/mlpack/core/optimizers/rmsprop/rmsprop.hpp

```diff
@@ -86,7 +90,7 @@ class RMSprop
   RMSprop(DecomposableFunctionType& function,
```

Comment:
Minor style issue (it was wrong in the first place); line-up parameters with DecomposableFunctionType.